.github/ISSUE_TEMPLATE/plugin-ownership-transfer.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Plugin Ownership Transfer
+about: Propose a plugin ownership transfer to the spinnaker-plugins organization. For more information, please see the documentation for [community donated plugins](https://github.com/spinnaker/governance/blob/master/sig-platform/community-donated-plugins.md)
+title: 'Plugin Transfer: <plugin name>'
+labels: committee/technical-oversight
+assignees: ajordens, ethanfrogers, ezimanyi, plumpy, robzienert
```

Comment:
Can this be a link to the document that has the current TOC membership in it?  That way, we do not need to remember to update this template when TOC membership changes.  If that doesn't work, then consider just putting in a prose description that the PR should be assigned to the current members of the TOC.

Reply:
> Can this be a link to the document that has the current TOC membership in it?

+1, if possible

Reply:
I can try to change it to reference `spinnaker/toc` - this part of the doc is for configuring GitHub, so it won't be something people see unless they're looking at the file - this  `assignees` block will auto-assign the issue to these members.

Reply:
+1 to `spinnaker/toc` to start with? If we end up with too much noise, we can reduce if needed?

Reply:
+1 I'm just not sure if groups are supported in this assignees block. We'll find out once it's merged!